### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by b-rage**

### CODE REVIEW

File: staff/gianluca-bragaglia/app-template/about.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    <nav>
+        <li><a href="home.html">Home</a></li>  
+        <li><a href="news.html">News</a></li> 
+        <li><a href="about.html">About</a></li> 
+        <li><a href="contact.html">Contact</a></li> 
+        <li><a href="profile.html">Profile</a></li> 
+    </nav>
+    <main>
+        <h2>Our Company</h2>
+        <article>
```
Comment (vior1992) [inline staff/gianluca-bragaglia/app-template/about.html]: Maybe is not necessary use this "article" here?

File: staff/gianluca-bragaglia/app-template/about.html [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+<body>
+    <header>
+        <h1>My App</h1>
+    </header>
+    <nav>
+        <li><a href="home.html">Home</a></li>  
+        <li><a href="news.html">News</a></li> 
+        <li><a href="about.html">About</a></li> 
+        <li><a href="contact.html">Contact</a></li> 
+        <li><a href="profile.html">Profile</a></li>
```
Comment (aleixcam) [inline staff/gianluca-bragaglia/app-template/about.html]: the ```<li>``` tags cannot be used  standalone, may you need to integrate them inside a ```<ul>```

File: staff/gianluca-bragaglia/app-template/contact.html [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Contact</title>
+</head>
+<body>
+    <header>
+        <h1>My App</h1>
+    </header>
+    <nav>
```
Comment (joel-molero) [inline staff/gianluca-bragaglia/app-template/contact.html]: You should move this ```<nav>```section into the bottom part of ```<header>```or the top part of ```<main>```, but do not leave it there floating.

File: staff/gianluca-bragaglia/app-template/contact.html [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+                    </p>
+                    <p>
+                      <button>Send Message</button>
+                    </p>
+                </form>
+        </section>                
+    </main>
+    <footer>
+        <address>
+                My App<br/>
```
Comment (joel-molero) [inline staff/gianluca-bragaglia/app-template/contact.html:48]: For better semantics, you should use ```<p>```opposed to ```<br>```. Just like in your "Contact us" form.

**Review by joel-molero — CHANGES_REQUESTED** [review]

File: staff/gianluca-bragaglia/app-template/about.html (hunk 1 above)
Comment (b-rage) [reply staff/gianluca-bragaglia/app-template/about.html]: Fixed

File: staff/gianluca-bragaglia/app-template/about.html (hunk 2 above)
Comment (b-rage) [reply staff/gianluca-bragaglia/app-template/about.html]: fixed

File: staff/gianluca-bragaglia/app-template/contact.html (hunk 4 above)
Comment (b-rage) [reply staff/gianluca-bragaglia/app-template/contact.html:48]: fixed

**Pull request merged**